### ISSUE TITLE
Transition `rke-tools` to `bci-busybox`

### ISSUE THREAD

**Issue by pjbgf** [rancher/rancher#46099]
Relates to https://github.com/rancher/rke-tools/pull/186.

**Comment by Sahota1225** [comment]
waiting for RKE1 October patches

**Comment by snasovich** [comment]
Moving to the next milestone as following merge of https://github.com/rancher/rke-tools/pull/186 there were many regressions and we will need to revisit it when we have a bit more time till the release.

**Comment by chiukapoor** [comment]
After migrating to bci-busybox in rke-tools, we added NGINX support by manually copying the NGINX binary and configuration from the official NGINX Docker image, as the NGINX package available via zypper is outdated (nginx/1.21.5).

While testing the November patches with bci-busybox migration, nginx-proxy container failed with missing user and files issue
```
12-03T07:54:31Z nov-patch-w-1 confd[7]: INFO Backend set to env
2024-12-03T07:54:31Z nov-patch-w-1 confd[7]: INFO Starting confd
2024-12-03T07:54:31Z nov-patch-w-1 confd[7]: INFO Backend source(s) set to 
nginx: [alert] could not open error log file: open() "/var/log/nginx/error.log" failed (2: No such file or directory)
2024/12/03 07:54:31 [emerg] 12#12: getpwnam("nginx") failed
```
We tried adding the missing user but the same is failing to get created in the docker image. 
```
RUN groupadd -g 101 nginx && \
    useradd -u 101 -r -g nginx -d /var/cache/nginx -s /sbin/nologin -c "nginx user" nginx
```

For now we will revert the changes and explore a more stable alternative for integrating NGINX into bci-busybox

**Comment by snasovich** [comment]
Closing as "won't do" per the above as we keep running into regressions.
